packages/terra-form-select/src/search/Frame.jsx

```diff
@@ -510,18 +510,24 @@ class Frame extends React.Component {
    * Falls back to the string 'Search' if no label is provided
    */
   ariaLabel() {
-    const { ariaLabel, disabled, intl } = this.props;
+    const {
+      ariaLabel, disabled, intl, placeholder,
+    } = this.props;
 
     const defaultAriaLabel = intl.formatMessage({ id: 'Terra.form.select.ariaLabel' });
     const dimmed = intl.formatMessage({ id: 'Terra.form.select.dimmed' });
+    const isChrome = navigator.userAgent.indexOf('Chrome') !== -1;
```

Comment:
why are we making chrome specific fix. I can see place holders are not being read on Safari browser as well.

Reply:
I was using Safari 16.5 version hence not able to repro the issue, in latest 17.1 version the issue is reproducible, I have ensured uniform functionality across all browsers now.